tests/acceptance/course-page/start-course-test.js

```diff
@@ -57,20 +57,20 @@ module('Acceptance | course-page | start-course', function (hooks) {
 
     assert.strictEqual(currentURL(), '/courses/dummy/introduction', 'current URL is course page URL');
 
-    let baseRequestsCount = [
-      'fetch courses (catalog)',
-      'fetch repositories (catalog)',
-      'fetch languages (catalog)',
-      'fetch leaderboard entries (course overview page)',
-      'refresh course (course overview page)',
-      'fetch courses (course page)',
-      'fetch repositories (course page)',
-      'fetch leaderboard entries (course page)',
-      'fetch languages (request language button)',
-      'fetch course language requests (request language button)',
-    ].length;
-
-    assert.strictEqual(apiRequestsCount(this.server), baseRequestsCount, `expected ${baseRequestsCount} requests`);
+    let expectedRequests = [
+      '/api/v1/repositories',
```

Comment:
@Arpan-206 let's still add comments so we know where these are coming from. The point of these request-counting tests is to ensure that we aren't unintentionally loading data twice, or loading more data than necessary. This can happen when using things like `query` (doesn't get cached) instead of `findAll` (does get cached)